File: Java/lab07/src/pessoas/Pessoa.java
package pessoas;

public class Pessoa {
    private String nome;
    private String endereco;

    public Pessoa() {
        super();
    }

    public Pessoa(String nome, String endereco) {
        this.nome = nome;
        this.endereco = endereco;
    }

    public void imprimir(){
        System.out.println("Nome: " + this.nome);
        System.out.println("Endereço: " + this.endereco);
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getEndereco() {
        return endereco;
    }

    public void setEndereco(String endereco) {
        this.endereco = endereco;
    }
}

File: C++/lab09/Pessoa.cpp
#include "Pessoa.hpp"

using namespace std;

Pessoa::Pessoa() {}

Pessoa::Pessoa(string nome, string cpf) {
    this->nome = nome;
    this->cpf = cpf;
}

string Pessoa::getNome() {
    return this->nome;
}

void Pessoa::setNome(string nome) {
    this->nome = nome;
}

string Pessoa::getCpf() {
    return this->cpf;
}

void Pessoa::setCpf(string cpf) {
    this->cpf = cpf;
}